eml/components/courses/section/SectionCard.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { View, Text, Pressable, Image, TouchableOpacity } from 'react-native';
```

Comment:
Update to new Text component (won't comment on further files, but please change this for all 😄)